content/Maximo/VISUAL-INSPECTION-ACTION.md

```diff
@@ -0,0 +1,214 @@
+2. **IBM Maximo Application Suite** log in page opens.
+3. Enter MVI **Username** and click **Continue**
+4. Enter MVI **Password** and click **Log in**
+5. Once logged in, on the MAS Suite navigator screen, click on the **Visual Inspection tile**. 
+
+![](_attatchments/mvi-action/MVI-action-detection-Step1.2.png)
+
+
+### Step 2: Create a Dataset for Model Training in Maximo Visual Inspection
+
+**What is an MVI dataset, and why do you need to create one?**
+
+As a first step towards building your AI inspection model, you must create a dataset in MVI containing videos of twisting on and off the caps of water bottles, shaking water bottles, and squeezing water bottles, to teach the model what each class looks like. 
+
+Data is at the heart of machine learning (ML). In the case of vision-based models, this data takes the form of images or videos. Built-in MVI ML models intelligently detect and learn patterns in the uploaded dataset and analyze this data repeatedly until it can discern distinctions and ultimately recognizes objects of interest in the images or videos. For example, to train a computer to recognize automobile tires, you need to feed it with vast quantities of tire images and tire-related items to learn the differences and identify a tire, especially one with defects. 
+For this use case model, you will create a dataset with a limited number of water bottle videos taken by yourself for brevity and accessibility; however, in real world scenarios, a computer vision model would require many more videos and processing time than this demonstration offers. 
+
+In this demo, you manually upload saved videos from your local machine to the MVI application dashboard, however, built-in image capture modes in MVI Mobile and MVI Edge applications (parts of the MVI solution) can also be used to collect and copy images and videos automatically into the MVI application. Now it's time to start creating a dataset. 
+
+**Actions:**
+
+1. Click **Get started** on the **MVI Welcome to IBM Maximo Visual Inspection** screen.
+
+![](_attatchments/mvi-action/MVI-action-detection-Step2.1.png)
+
+2. The **Data sets**window opens.
+
+3. On the MVI **Data sets** screen, Click the **blue Create +** icon in the top right corner.
+
+![](_attatchments/mvi-action/MVI-action-detection-Step2.3.png)
+
+4. Enter **“Water Bottle Action Detection Dataset [Your Initials]”** for the data set name field in the **Create data set** window. For data set type, select *Training*.
+
+***IMPORTANT: Please make sure to add your initials to the name of the data sets, training models and deployed models as it will be helpful to keep track of your assets since the demo is conducted in a shared environment.*
+
+![](_attatchments/mvi-action/MVI-action-detection-Step2.4.png)
+
+5. Click **Create**. 
+ 
+6. Click on the newly created dataset and click on the blue hyperlink in the top left corner of the **Upload files** box that will allow you to upload files. 
+
```

Comment:
Should add a section that talks about downloading, unzipping, and using provided videos if they wish to use those.